distro.py

```diff
@@ -22,6 +24,33 @@
     (os.path.normcase("arelle/locale"), "locale"),
     (os.path.normcase("arelle/plugin"), "plugin"),
 ]
+
+# Pip-installed plugins cannot be discovered in frozen builds because
+# the `ast` library can't parse `__pluginInfo__` from compiled .pyc files
+# Instead, pip-intalled plugins can be copied directly into the plugins folder
+
+# Built list of absolute paths to available pip packages
+packageDirectories = []
+sitePackagesDirectories = site.getsitepackages()
+for sitePackagesDirectory in sitePackagesDirectories:
+    packageDirectories.extend([os.path.join(sitePackagesDirectory, x) for x in os.listdir(sitePackagesDirectory)])
+
+# For each entry point discovered, find the pip package containing it
+# and stage for copying into the plugins build directory
+if sys.version_info < (3, 10):
+    entryPoints = [e for e in entry_points().get('arelle.plugin', [])]
+else:
+    entryPoints = list(entry_points(group='arelle.plugin'))
+for entryPoint in entryPoints:
+    pluginUrl = entryPoint.load()()
+    pluginDirectory = None
+    for packageDirectory in packageDirectories:
+        if pluginUrl.startswith(packageDirectory):
+            pluginDirectory = packageDirectory
+            break
+    assert pluginDirectory, f"Corresponding package could not be found for plugin path: {pluginUrl}"
+    includeFiles.append((pluginDirectory, os.path.join('plugin', os.path.basename(pluginDirectory))))
```

Comment:
Looks like this doesn't work on the windows build.

Could you add an assertion to the build workflows (Linux, macOS, and Windows) to confirm `iXBRLViewerPlugin` is in the plugins directory?

Reply:
Added on this PR: https://github.com/Arelle/Arelle/pull/1006